packit/status.py

```diff
@@ -136,6 +137,32 @@ def get_builds(self,) -> Dict:
         builds = {b["nvr"].rsplit(".", 1)[1]: b["nvr"] for b in reversed(builds_l)}
         return builds
 
+    def get_copr_builds(self, number_of_builds: int = 5) -> List:
+        """
+        Get the copr builds of this project done by packit.
+        :return: list of builds
+        """
+        client = CoprClient.create_from_config_file()
+
+        projects = [
+            project.name
+            for project in reversed(client.project_proxy.get_list(ownername="packit"))
+            if project.name.startswith(
+                f"{self.up.local_project.namespace}-{self.up.local_project.repo_name}-"
```

Comment:
sadly this doesn't work for me because my origin is my fork, not the upstream repo; hence it tries to look for TomasTomecek-packit instead of packit-service-packit

I'm gonna open an issue to solve this

Reply:
Should I wait for solving of #586 or changing of the condition to ```self.up.local_project.repo_name in project.name``` will be enough?